### DISCUSSION TITLE
Binding to copy google search results

### DISCUSSION THREAD

**Discussion by txtyash** [gdh1995/vimium-c#830] [Q&A]
# On running a google search we usually see results like:
![image](https://user-images.githubusercontent.com/85027668/209448553-e2a210bf-3a38-4a68-93b9-a40322f8af1a.png)

# Off of which this part is what most people are concerned with:
![image](https://user-images.githubusercontent.com/85027668/209448543-1ae0d922-1e73-4057-b416-ec8c3fe2cf2a.png)

# Can there be a binding such as `yg` to quickly copy this?

**Comment by gdh1995** [comment]
You may use `LinkHints.click` to let LinkHints select an element directly, and in this case the below command should work:
```
map xxx LinkHints.click mode="copy-text" match="[data-attrid=\"wa:/description\"]"
```
